Wei/View/Send/SelectAmount/SelectAmountViewModel.swift

```diff
@@ -48,56 +48,68 @@ final class SelectAmountViewModel: InjectableViewModel {
         // NOTE: Deal with fixed tx fee
         let txFee = Wei(Gas.normal.gasLimit * Gas.normal.gasPrice)
         
+        // fiatTxFeeAction converts specified tx fee to fiat price.
+        let fiatTxFeeAction = input.viewWillAppear.flatMap { [weak self] _ -> Driver<Action<Price>> in
+            guard let weakSelf = self else {
+                return Driver.just(Action.succeeded(Price(price: "0", currency: nil)))
+            }
+            let source = weakSelf.rateRepository.convertToFiat(from: txFee.description).asObservable()
+            return Action.makeDriver(source)
+        }
+        
+        // stripe the decimal amount from tx fee.
+        let fiatTxFee = fiatTxFeeAction.elements
+            // only use number before the decimal poin.
+            // for exsample 1 for 1.2345
+            .map { String($0.price.split(separator: ".").first ?? "") }
```

Comment:
[IMO]should ceil data
For example fiatTexFee is 1.2345, fiatBalance is 10.
The availableBalance should be 8.
In case of that logic, the available balance is 9 yen 🤔 ...